src/data/community-events.json

```
@@ -1,12 +1,14 @@
 [
   {
-    "title": "Road to Devcon（Shenzhen）",
-    "to": "https://ethsz.openbuild.xyz/",
-    "sponsor": null,
-    "location": "Shenzhen, China",
-    "description": "Preheating activities for DEVCON 7 in the China region.Developers and communities, let us gather at Devcon！",
-    "startDate": "2023-10-21",
-    "endDate": "2023-10-21"
+    "title": "Road to Devcon（Hongkong）",
+    "to": "https://devcon-hk.openbuild.xyz/",
+    "sponsor": null,
+    "location": "Hongkong, China",
+    "description": "Preheating activities for DEVCON 7 in the China region.Developers and communities, let us gather at Devcon！
+  Location: University of Hong Kong (HKU) ,
+  Time: October 25, 1:00 pm to 5:00 pm.",
```

Thanks for the PR @Mirror-Tang. 

No need to include the location and time in the description here.

```suggestion
    "description": "Preheating activities for DEVCON 7 in the China region. Developers and communities, let us gather at Devcon!",
```

---

I have deleted the time and location in the description, please check the latest version in time, thank you！